src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -138,6 +139,31 @@ private function registerMetadataConfiguration(ContainerBuilder $container, XmlF
     }
 
     /**
+     * Registers the OAuth2 configuration.
+     *
+     * @param ContainerBuilder $container
+     * @param array            $config
+     * @param XmlFileLoader    $loader
+     */
+    private function registerOAuth2Configuration(ContainerBuilder $container, array $config, XmlFileLoader $loader)
```

Comment:
I suggest to use `oauth` everywhere instead of `oauth2`. We don't include the version for other formats such as Hydra, Swagger and so on.